library.html

```diff
@@ -10,7 +10,7 @@
 
 	<body>
 	
-		<div class = "wrapper">
+		<div class = "wrapper2">
```

Comment:
The library page is now under wrapper2 which has columns and rows designed for the page.